packages/rpc-core/src/bundle.ts

```diff
@@ -4,8 +4,8 @@
 import type { Observer } from 'rxjs';
 import type { ProviderInterface, ProviderInterfaceCallback } from '@polkadot/rpc-provider/types';
 import type { StorageKey, Vec } from '@polkadot/types';
-import type { Hash } from '@polkadot/types/interfaces';
-import type { AnyJson, Codec, DefinitionRpc, DefinitionRpcExt, DefinitionRpcSub, Registry } from '@polkadot/types/types';
+import type { BlockNumber, Hash } from '@polkadot/types/interfaces';
```

Comment:
```suggestion
import type { Hash } from '@polkadot/types/interfaces';
```

As per next comment.